samples/client/petstore/java/okhttp-gson/src/test/java/io/swagger/client/api/PetApiTest.java

```diff
@@ -269,6 +264,7 @@ public void testFindPetsByStatus() throws Exception {
     }
 
     @Test
+    @Ignore
```

Comment:
This endpoint is deprecated on petstore.swagger.io and was returns a 500 status.